src/itemdb.d

```diff
@@ -91,6 +81,34 @@ final class ItemDatabase
 		deleteItemByIdStmt = db.prepare("DELETE FROM item WHERE driveId = ? AND id = ?");
 	}
 
+	void createTable(int dbVersion)
```

Comment:
`dbVersion` is never used in the method at all - either we need to remove the parameter or pass `itemDatabaseVersion` to it perhaps on line 60?

Reply:
Ahh .. yes .. wrong copy & paste there.